src/FSLibrary/StellarJobExec.fs

```diff
@@ -179,6 +179,11 @@ type StellarFormation with
         : Map<string, bool> =
         let jst = new JobStatusTable()
         let mutable moreJobs = true
+
+        // We have seen sporadic http exceptions being thrown in the while loop below,
+        // so this is an attempt to see if we can just ignore the exceptions.
+        let mutable httpRequestErrorsAllowed = 5
```

Comment:
How about naming this as numOfRetriesAllowed?

Reply:
I renamed to `numOfHttpRetriesAllowed` to make the retry point more clear.